src/components/Tool/Appeal/AppealsContext/AppealsContext.tsx

```diff
@@ -70,6 +77,8 @@ export interface AppealsType
   committedCountQueryResult: ReturnType<typeof useContactsCountQuery>;
   givenCountQueryResult: ReturnType<typeof useContactsCountQuery>;
   receivedCountQueryResult: ReturnType<typeof useContactsCountQuery>;
+  refreshFlowsView: boolean;
+  seRefreshFlowsView: Dispatch<SetStateAction<boolean>>;
```

Comment:
I see a bunch of other occurrences with this typo. I'm not going to comment on them all, but try a global find/replace to fix them all.


```suggestion
  setRefreshFlowsView: Dispatch<SetStateAction<boolean>>;
```